odoo/models.py

```diff
@@ -749,7 +749,7 @@ def to_xid(record_id):
             )))
             for r in missing
         )
-        fields = ['module', 'model', 'name', 'res_id']
+        fields = ['module', 'model', 'name', 'res_id', 'create_date']
```

Comment:
Shouldn't the `write_date` field be set too?

Reply:
@luisg123v #50661 subsumes this PR and adds an SQL-level default value to both `create_date` and `write_date` (on `ir.model.data`) so it handles both, and a few other improvements.

Reply:
:+1: